src/main/java/com/patchworkmc/manifest/converter/ModManifestConverter.java

```diff
@@ -105,4 +110,26 @@ private static JsonArray convertAuthorsList(String authors) {
 
 		return array;
 	}
+
+	private static JsonObject getDependencies(ModManifest manifest, ModManifestEntry mod) {
+		JsonObject depends = new JsonObject();
+		Map<String, List<ModManifestDependency>> dependencyMap = manifest.getDependencyMapping();
+		dependencyMap.get(mod.getModId()).forEach(c -> {
+			if (c.isMandatory()) {
+				depends.addProperty(c.getModId(), c.getVersionRange());
```

Comment:
Should make sure that the version format matches what Fabric expects.

Reply:
Forge says they use "Maven style" and Fabric says they use "SemVer 2.0.0 style", which as far as I can tell is the same thing

Reply:
This doesn't look like valid SemVer to me: `versionRange="[24,)"`